rs/registry/canister/canister/canister.rs

```diff
@@ -180,6 +180,19 @@ fn canister_init() {
             registry.maybe_apply_mutation_internal(mutation_request.mutations)
         });
     recertify_registry();
+
+    #[cfg(feature = "test")]
+    {
+        use registry_canister::flags::temporary_overrides::{
+            test_set_swapping_enabled_subnets, test_set_swapping_status,
+            test_set_swapping_whitelisted_callers,
+        };
+
+        println!("{LOG_PREFIX}canister_init: Overriding swapping flags");
+        test_set_swapping_status(init_payload.is_swapping_feature_enabled);
```

Comment:
These don't seem necessary to me.  You could accomplish the same thing by setting `static IS_NODE_SWAPPING_ENABLED: Cell<bool> = const { Cell::new(false) };` to instead be `static IS_NODE_SWAPPING_ENABLED: Cell<bool> = const { Cell::new(cfg!(feature = "test") };`